keyboards/cutie_club/wraith/wraith.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2019 Amber Holly
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
If this macro supports every possible switch position on the PCB, it should be named `LAYOUT_all` and the default keymap should use it, even if it's impossible to build the hardware to use every position.

Additional macros representing different physical layouts (like your ANSI and ISO variants, or Split Backspace vs. 2u Backspace) should be added, if the hardware supports multiple layouts.